src/main/java/no/finn/unleash/util/UnleashConfig.java

```diff
@@ -26,6 +26,7 @@
     private final String instanceId;
     private final String sdkVersion;
     private final String backupFile;
+    private final String projectName;
```

Comment:
So we have no intention of a SDK to be able to be part of "multiple projects"?

Reply:
After discussion offline, for now this is kept as a String, not a collection of Strings. However, we've made the UnleashConfig constructor private, so we can change this without making another breaking change if we see the need for adding support for multiple projects.